karapace/protobuf/protobuf_to_dict.py

```diff
@@ -0,0 +1,336 @@
+# -*- coding:utf-8 -*-
```

Comment:
this module is missing typing information.

Reply:
it is 3rd party module which with minor fixes to be compliant to our project.